parm/archive/gdas.yaml.j2

```diff
@@ -79,6 +92,20 @@ gdas:
         - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/{{ head }}snowstat.tgz"
             {% endif %}
 
+        # Snow surface data
+        {% if DO_JEDISNOWDA %}
+        {% for itile in range(1,7) %}
+        # Snow analysis is 3dvar
+        - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/snowinc.{{ cycle_YMD }}.{{ cycle_HH }}0000.sfc_data.tile{{ itile }}.nc"
+        - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/{{ cycle_YMD }}.{{ cycle_HH }}0000.sfc_data.tile{{ itile }}.nc"
+        {% endfor %}
+        {% endif %}
+
+        # Snow configuration yaml
+        {% if DO_JEDISNOWDA %}
+        - "{{ COMIN_CONF | relpath(ROTDIR) }}/{{ head }}snowanlvar.yaml"
+        {% endif %}
```

Comment:
Suggest to put analysis states including `snowstat`, `configuration yaml` and `snow surface data` into a single `{% if DO_JEDISNOWDA %}` and `{% endif %}` block.